### PR TITLE
[HOTFIX] Fixing crash on startup of tuplex.Context() due to missing AWS credentials

### PR DESCRIPTION
**Pull request by LeonhardFS**
When no AWS credentials are specified (i.e. via `aws configure` or using environment variables `AWS_SECRET_ACCESS_KEY`/`AWS_ACCESS_KEY_ID`) a segfault is produced due to a buggy Boost.Python exception translation. 

- This commit updates CI scripts to add a pip package test
- test.pypi.org packages are now created always
- exceptions during Context creation are handled without relying on Boost.Python

Version should be pushed to 0.3.0.post1 or better even 0.3.1 to avoid frustration of pulling a segfaulting package.

### CODE REVIEW

File: tuplex/python/src/PythonBindings.cc [hunk 1]
```diff
@@ -35,7 +35,7 @@ PYMODULE {
     // std::cout<<"init done"<<std::endl;
 
 
-    register_exception_translator<tuplex::PythonException>(&tuplex::translateCCException);
+    //register_exception_translator<tuplex::PythonException>(&tuplex::translateCCException);
```
Comment (ms705) [inline tuplex/python/src/PythonBindings.cc:38]: Remove or leave a comment as to why this is commented out (so we don't forget in the future).

**Review by ms705 — APPROVED** [review]
LGTM

File: tuplex/python/src/PythonBindings.cc (hunk 1 above)
Comment (LeonhardFS) [reply tuplex/python/src/PythonBindings.cc]: done.

**Comment by niviksha** [comment]
Thanks for the quick response! I can confirm this happens on the straight pip install on Linux too, assuming the fix will apply to that platform as well.

**Pull request merged**